### ISSUE TITLE
no such file or directory lv

### ISSUE THREAD

**Issue by asarchami** [syl20bnr/spacemacs#12124]
I keep getting (file-missing "Cannot open load file" "No such file or directory" "lv").
Everything was ok till I decided to install a clean spacemacs! no I get this. I have `lv` installed on my os.
I also manually installed package `lv`.

OS: Ubuntu 18.10
emacs version: 26.1

**Comment by smile13241324** [comment]
Strange I can have a look but first adjust your issue to match the official issue template please. There please include a more detailed explanation when you are getting this error and whether emacs stops loading or just a random package is broken.

Last but not least please load a default dotfile and say whether the error still occurs.

**Comment by LdBeth** [comment]
Having the same issue here after update all packages.
The package requires `lv` is `hydra`, possibly an upstream issue. Adding `lv` to `dotspacemacs-additional-packages` seems don't work. If you don't use it simply exclude it could be a work around.

---
#### System Info :computer:
- OS: darwin
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 18a998b67)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(helm auto-completion better-defaults emacs-lisp scheme
      (haskell :variables haskell-completion-backend 'dante haskell-enable-hindent t)
      agda latex spell-checking)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS MODULES THREADS

**Comment by ginhton** [comment]
I met the same problem.
And test under ubuntu 16.04 and windows 10. All get the same error.
What I have done is just clone repo and run emacs.

**Comment by Congrammers** [comment]
also having the same problem; tried
master
LdBeth/spacemacs
release-0.200
and still having the issue.

**Comment by LdBeth** [comment]
@Congrammers check out `develop` branch from my PR. Try start from a new `.spacemacs` file, it should work fine.

**Comment by bmag** [comment]
This is an [upstream issue](https://github.com/abo-abo/hydra/issues/323) in Hydra and/or Melpa (mismatch between Hydra's code and Hydra's recipe). Hopefully it will be sorted out soon, but until then you can apply the changes from https://github.com/syl20bnr/spacemacs/pull/12125/files as a workaround (don't forget to undo the changes once the issue is resolved).

An alternative workaround, if you don't want to change Spacemacs code, mentioned by @LdBeth:

> downloaded a copy of lv.el from https://github.com/abo-abo/hydra/blob/master/lv.el and load in dotspacemacs/user-init

Adding `lv` as an additional package may not work because Hydra is loaded in the bootstrap phase.

**Comment by duianto** [comment]
This seems to have been fixed upstream now. Thanks bmag.
Declare dependency on separate lv package https://github.com/abo-abo/hydra/commit/480ae6313763a6af6dab85ed629aff2e2037bef0

There's a `hydra` update, the package description `SPC h d p` `hydra` `RET` shows:
>    Version: 20190405.1015
    Summary: Make bindings that stick around.
   Requires: cl-lib-0.5, lv-0

**Comment by asarchami** [comment]
Thanks for the help.

**Comment by yuhan0** [comment]
Looks like this issue was fixed upstream and can be closed: #12125
